app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,24 @@
+class RecipesController < ApplicationController
```

Comment:
- [ ] It will be great to implement all the crud operations except the `update`. This is a major requirement for this project to be approved. What I mean by this is that a user should be able to create his own recipes. In your implementation, you are seeding some recipes for just the users that you have described. kindly create a form for creating a new recipes. ⬇️ 
![Screenshot 2023-05-26 at 15 17 55](https://github.com/ram1117/recipe-app/assets/32361963/7605acd0-1ee8-41a2-b694-35740eaa44e3)